### PR TITLE
[RHMAP 5251] FHC Command to import/export forms

### PR DESCRIPTION
**Pull request by paolobueno**
Contains the content in PRs #192 and #193, also added unit test coverage for not overwriting files on `export` and some linting style fixes and version bump.

ping @corinnekrych @vchepeli @wei-lee @wtrocki @pb82

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* A component owner should review your change (we've tagged these above)
* This change should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* Ensure JIRA is up to date (e.g. deployment notes, tester notes, fix version, release notes)
* There should be an associated fhcap PR with this change - please provide a link to this
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.

**Comment by paolobueno** [comment]
Squashed commits

**Comment by wei-lee** [comment]
I have tried both commands with the latest export/import implementation on the server side, everything works fine. 
Just one small comment:  perhaps print out a message on the console when the export/import is finished to say something like "The operation is completely successfully (or failed)" to make it obvious. At the moment, nothing is printed out when the operation is done, so user might be confused.

**Comment by paolobueno** [comment]
@wei-lee Thanks for the revision. I'm currently trying to use `log.info()` but it seems that unlike it is written on the file, default log level is below info. I noticed there is a lone `console.log` call inside the download utility function for exporting, inside the `request` module, which I think is kind of bad, since we don't want output coming from calls deeper within utility functions.

How do you recommend implementing the extra output?

**Comment by wei-lee** [comment]
@paolobueno can we just add a console.log around https://github.com/paolobueno/fh-fhc/blob/RHMAP-5251/lib/cmd/fh3/appforms/forms/export.js#L33, and similarly to the import function.

File: lib/cmd/fh3/appforms/forms/export.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    params.url = this.url;
+    params.method = this.method;
+    params.output = params.file;
+
+    if (path.extname(params.file) !== ".zip") {
+      return cb("Expected the output file to have a .zip extension");
+    }
+
+    fs.stat(params.file, function(err) {
+      if (!err) {
```
Comment (vchepeli) [inline lib/cmd/fh3/appforms/forms/export.js]: why !err?

**Comment by vchepeli** [comment]
LGTM

File: lib/cmd/fh3/appforms/forms/import.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        return cb(new Error('File ' + path + " not found"));
+      }
+      if (!stats.isFile()) {
+        return cb(new Error(path + " is not a file"));
+      }
+      upload(path, url, function(err) {
+        if (err) {
+          return cb(err);
+        }
+        console.log('File ' + path + ' uploaded successfully');
```
Comment (fh-qube) [inline lib/cmd/fh3/appforms/forms/import.js]: MAJOR Remove this logging statement. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**